src/cmd-build

```diff
@@ -262,13 +277,18 @@ cat > tmp/meta.json <<EOF
  "coreos-assembler.vm-iso-checksum": "${vm_iso_checksum}",
  "coreos-assembler.container-config-git": $(jq -M '.git' ${PWD}/coreos-assembler-config-git.json),
  "images": {
-    "qemu": { "path": "${img_qemu}",
-              "sha256": "${img_qemu_sha256}",
-              "size": "${img_qemu_size}"
-    }
- }
-}
 EOF
+# Add our images
+first=true
+(for image in "${!images[@]}"; do
+     path=${images[${image}]}
+     checksum=$(sha256sum ${path} | cut -f 1 -d ' ')
+     size=$(stat -c '%s' ${path})
+     if ${first}; then first=false; else echo ','; fi
+     echo '"'${image}'": {"path": "'${path}'", "sha256": "'${checksum}'", "size": "${size}"}'
+ done
+ echo "}}"
+) >> tmp/meta.json
```

Comment:
Hmm, how about playing the same trick as with `tmp/cosa-image.json`? Feels odd to write out `tmp/meta.json` initially invalid.

Actually, related to this, would be nice if this supported e.g. `coreos-assembler build none` if one only wants to build an OSTree (in which case, there wouldn't be an `"images"` key at all in the `meta.json`). I've definitely wanted that in the past when just hacking on the OSTree bits. We can think about that later though (but it motivates a bit more separating out the `"images"` injection here).

Reply:
OK, the first part is done.